src/Catty/Parser&Serialializer/New Parser & Serializer/Serializer/Context/CBXMLSerializerContext.h

```diff
@@ -58,11 +57,16 @@
 @property (nonatomic, strong) NSMutableDictionary *projectUserListNamePositions;
 
 //------------------------------------------------------------------------------------------------------------
-// ressources data used while traversing the tree
+// resources data used while traversing the tree
 //------------------------------------------------------------------------------------------------------------
 // TODO: refactor this later: remove brickList here and dynamically find brick in scriptList. maybe scripts should be referenced in bricks as well!!
 @property (nonatomic, strong) NSMutableArray *brickList; // contains all bricks (used only by serializer)
 
+@property (nonatomic, strong) Project *project;
+
+- (instancetype)initWithProject:(Project *)project;
+- (instancetype)init NS_UNAVAILABLE;
```

Comment:
Since `new` is not unavailable yet, this class can still be initialized without a project.